docs/source/azure/configuration/authentication.rst

```diff
@@ -112,6 +112,32 @@ You must set `AZURE_SUBSCRIPTION_ID` as with the other authentication types.
     AZURE_SUBSCRIPTION_ID=subscriptionId
     AZURE_CLIENT_ID=clientId
 
+Azure Key Vault Integration
+---------------------------
+
+If you run Custodian inside Azure VM, AKS, ACI or Azure Function you can leverage Azure Key Vault to store
+Service Principal credentials. You can store json-formatted authentication file as a Key Vault secret.
+It will be used to extend your local authentication options. It is recommended to configure Subscription ID
+and Tenant ID locally and store SP credentials in Azure Key Vault.
```

Comment:
It is recommended to configure the **Subscription Id** and **Tenant ID** locally and store the Service Principal credentials in Azure Key Vault.

Highlight the keywords - SubId, TenantId